lib/mailer_previews/faculty_notifications_mailer_preview.rb

```diff
@@ -11,7 +11,7 @@ def open_access_reminder
     old_fake_pubs = [pub1, pub2]
     new_fake_pubs = [pub3]
 
-    FacultyNotificationsMailer.open_access_reminder(fake_user, old_fake_pubs, new_fake_pubs)
+    FacultyNotificationsMailer.open_access_reminder(fake_user, old_fake_pubs + new_fake_pubs)
```

Comment:
Minor thing, but this could be cleaned up a little bit to get rid of the local assignments above and the array concatenation.

i.e.
```ruby
FacultyNotificationsMailer.open_access_reminder(fake_user, [pub1, pub2, pub3])
```